client/js/commands/search.js

```diff
@@ -4,15 +4,19 @@ import store from "../store";
 import {router} from "../router";
 
 function input(args) {
-	router.push({
-		name: "SearchResults",
-		params: {
-			id: store.state.activeChannel.channel.id,
-		},
-		query: {
-			q: args.join(" "),
-		},
-	});
+	if (!store.state.settings.searchEnabled) {
+		store.commit("currentUserVisibleError", "Search is currently not enabled.");
```

Comment:
I am not so keen on that one. The `/search` command is made for people who like to do everything with the keyboard. And these errors can only be removed by clicking on them (so the user has to reach the mouse). 

Maybe we could add a timeout functionality to these errors in general (or put a simple `setTimeout` in which we check if it's still the same error and if so remove the error (if we don't add a check we might remove another error which we definitely don't want))

Reply:
Applied a setTimeout for 5 seconds. That said, it would be inconsistent with other things (say, HTTP upload errors and so on), which stay forever until you click on them.

Reply:
To be fair, I think the best action here is that instead of showing an error, adding an information message on the currently open channel. Opinions? (cc @MaxLeiter)

Reply:
I agree currentUserVisibleError shouldn't be used for this. Make it a message so it functions more like responses to other commands